### PR TITLE
docs: align portal iam doc structure with trg 1

### PR DESCRIPTION
**Pull request by shahmargi12**
## Description

- **Restructured existing documents** according to the new TRG 1 folder hierarchy.
- Created a new README.md with links to the relevant files and subfolders, ensuring all documents are properly referenced and accessible.

Structure that followed in PR -

```
repository
└─── docs/
    └─── admin/
         └─── technical-documentation/
         └─── dev-process/
         └─── release-process/
         └─── known-knowns/
    └─── consultation/
         └─── portal-iam-helm.chart.md
         └─── charts/
         └─── environment/
         └─── workshops/
         └─── consultation.md
    └─── README.md

```
## Why

To align with TRG 1 

## Issue

#187 

## Checklist

Please delete options that are not relevant.

- [X] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-iam/blob/main/docs/technical%20documentation/14.%20How%20to%20contribute.md)
- [X] I have added copyright and license headers, footers (for .md files) or files (for images) 
- [X] I have performed a self-review of my changes
- [X] I have successfully tested my changes

### CODE REVIEW

**Comment by shahmargi12** [comment]
Hello @evegufy @MaximilianHauer 

I noticed that the quality gate is failing due to some issues in the YAML files. The only change I made related to these files is moving their folder location, without modifying the content itself.

Could you please take a look and help me resolve the issue? It seems like the errors might be triggered by the folder restructuring.

**Comment by evegufy** [comment]
> Hello @evegufy @MaximilianHauer
> 
> I noticed that the quality gate is failing due to some issues in the YAML files. The only change I made related to these files is moving their folder location, without modifying the content itself.
> 
> Could you please take a look and help me resolve the issue? It seems like the errors might be triggered by the folder restructuring.

@shahmargi12 thank you for the contribution!
Regarding sonarcloud: could you please add such a file https://github.com/eclipse-tractusx/portal-frontend/blob/main/.sonarcloud.properties where you exclude the docs directory? https://stackoverflow.com/questions/21323276/sonarqube-exclude-a-directory Thank you!

**Comment by shahmargi12** [comment]
@evegufy  I’ve updated the .sonarcloud.properties file to exclude the docs directory. You can find the details in this PR: [#1275](https://github.com/eclipse-tractusx/portal-frontend/pull/1275). 
Thank you!

**Comment by shahmargi12** [comment]
Hello @evegufy 
Apologies for the previous confusion. I've created a pull request (#222) for the required change in the portal-iam repository.
Let me know if you need any adjustments.

**Comment by evegufy** [comment]
> Hello @evegufy Apologies for the earlier confusion. I’ve updated the `sonarcloud.properties` file to ignore the .yaml file inside the docs folder.
> 
> Let me know if you need any further adjustments.

@shahmargi12 perfect, thank you! sorry, I didn't remember that there is already a `sonarcloud.properties` and the existing one could to be updated.

**Review by evegufy — CHANGES_REQUESTED** [review]
I had a look and as architecture and user documentation doesn't exist in this repo, I think it makes sense to restructure like this:

```
repository
└─── docs/
    └─── admin/
         └─── technical-documentation/
         └─── known-knowns/
         └─── dev-process/
         └─── release-process/
    └─── consultation/
         └─── portal-iam-helm.chart.md
         └─── charts/
         └─── environment/
         └─── workshops/
         └─── consultation.md
    └─── README.md

```

**Comment by shahmargi12** [comment]
> I had a look and as architecture and user documentation doesn't exist in this repo, I think it makes sense to restructure like this:
> 
> ```
> repository
> └─── docs/
>     └─── admin/
>          └─── technical-documentation/
>          └─── known-knowns/
>          └─── dev-process/
>          └─── release-process/
>     └─── consultation/
>          └─── portal-iam-helm.chart.md
>          └─── charts/
>          └─── environment/
>          └─── workshops/
>          └─── consultation.md
>     └─── README.md
> ```

@evegufy Thanks for the suggestion! I’ve restructured the documentation according to your recommendations

**Review by evegufy — APPROVED** [review]
looks really good, thank you @shahmargi12

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+The `charts` folder contains Helm charts for deploying the Portal IAM system. Refer to the following for detailed usage:
+
+- [Portal IAM Helm Chart](./consultation/portal-iam-helm-chart.md): Instructions on using the Helm chart for deploying Portal IAM on Kubernetes.
+
+### 2. [Workshops](./consultation/workshops/)
+
+The `workshops` folder includes practical resources for learning and deploying Portal IAM:
+
+- [Workshop 20230927](./consultation/workshops/workshop-20230927.md): Materials and steps covered during the September 27, 2023 workshop.
+- [Workshop 20231005](./consultation/workshops/workshop-20231005.md): Materials and steps from the October 5, 2023 workshop.
```
Comment (evegufy) [inline docs/README.md:60]:
```suggestion
The `workshops` folder includes minutes and topics for two workshops:

- [Workshop 20230927](./consultation/workshops/workshop-20230927.md): topics covered during the session of September 27, 2023 workshop.
- [Workshop 20231005](./consultation/workshops/workshop-20231005.md): topics covered during the session of October 5, 2023 workshop.
```

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+- [Known-Knowns](./admin/known-knowns/Known-Knowns.md): A list of known issues and solutions.
+
+### [Consultation Documentation](./consultation/)
+
+### 1. [Charts](./consultation/charts/)
+
+The `charts` folder contains Helm charts for deploying the Portal IAM system. Refer to the following for detailed usage:
+
+- [Portal IAM Helm Chart](./consultation/portal-iam-helm-chart.md): Instructions on using the Helm chart for deploying Portal IAM on Kubernetes.
```
Comment (evegufy) [inline docs/README.md:53]:
```suggestion
### 1. [charts](./consultation/charts/) and [environments](./consultation/environments/)

The `charts` and `environments` folder contain an **example** helm chart and configuration for a potential deployment improvement discussed during workshops.

- [Portal IAM Helm Chart](./consultation/portal-iam-helm-chart.md): instructions on using this **example** helm chart.
```

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+### 2. [Workshops](./consultation/workshops/)
+
+The `workshops` folder includes practical resources for learning and deploying Portal IAM:
+
+- [Workshop 20230927](./consultation/workshops/workshop-20230927.md): Materials and steps covered during the September 27, 2023 workshop.
+- [Workshop 20231005](./consultation/workshops/workshop-20231005.md): Materials and steps from the October 5, 2023 workshop.
+
+
+### 3. [Consultation Document](./consultation/consultation.md)
+- Key architectural consultation notes and decisions.
```
Comment (evegufy) [inline docs/README.md:64]:
```suggestion
### 3. [Consultation Document](./consultation/consultation.md)

- Consultation notes and decisions.
```

**Review by evegufy — CHANGES_REQUESTED** [review]
sorry I noticed that there's still a change needed

**Comment by shahmargi12** [comment]
> sorry I noticed that there's still a change needed

Thanks for catching that! I’ve made the change as requested. Let me know if there's anything else that needs updating.

**Review by evegufy — APPROVED** [review]

**Pull request merged**